.github/workflows/build.yaml

```diff
@@ -0,0 +1,120 @@
+name: Build
+
+on:
+  workflow_call:
+    inputs:
+      mode:
+        required: true
+        type: string
+        description:
+          mode as expected by `cc-utils/.github/workflows/prepare`
+          either of
+          - snapshot
+          - release
+
+jobs:
+  prepare:
+    uses: gardener/cc-utils/.github/workflows/prepare.yaml@master
+    with:
+      mode: ${{ inputs.mode }}
+
+  lint:
+    runs-on: ubuntu-latest
+    needs:
+      - prepare
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: latest
+
+      - name: Run ESLint
+        run: |
+          .ci/lint
+
+      - name: Run ESLint (SAST) and collect evidence
+        run: |
+          blobs_dir=blobs.d
+          mkdir -p $blobs_dir
+          eslint_logfile=eslint-sast.log
+          eslint_evidence="${blobs_dir}/eslint-sast.tar.gz"
+
+          .ci/lint eslint.sast.config.mjs 2>&1 | tee $eslint_logfile || true
```

Comment:
I think we should have our pipeline fail upon findings....

-> `set -euo pipefail` (first line)
-> drop || true